lib/kitchen/directions/bake_notes/bake_numbered_notes/main.rb

```diff
@@ -21,8 +25,14 @@ def self.bake_note_exercise(note:, exercise:, divider: ' ')
         return unless exercise.solution
 
         # bake solution in place
-        BakeNumberedExercise.bake_solution_v1(
-          exercise: exercise, number: note.first('.os-number').text.gsub(/#/, ''), divider: divider)
+        if suppress_solution
+          exercise.add_class('os-hasSolution')
```

Comment:
do we not add this class if there is a solution but `suppress_solution` is false?

Reply:
if there is a solution, `BakeNumberedExercise.bake_solution_v1` adds that class to the exercise. If the solution is suppressed, it apparently still needs the `os-hasSolution` class

Reply:
@jpslav @LostMojo I have made some changes (together with you suggestions JP) in this PR: https://github.com/openstax/kitchen/pull/181. Please look into it if you would like to.